api/config/test.exs

```diff
@@ -0,0 +1,28 @@
+import Config
+
+# Configure your database
+#
+# The MIX_TEST_PARTITION environment variable can be used
+# to provide built-in test partitioning in CI environment.
+# Run `mix help test` for more information.
+config :api, Api.Repo,
+  username: "postgres",
+  password: "postgres",
+  hostname: "localhost",
+  database: "performance_manager",
```

Comment:
i guess i'm still a little unclear on how the testing against a db is going to work in practice since its going to be seeded from another source. i would imagine we would keep the db name the same rather than have the process that generates it use a different one?